packages/projects-docs/pages/learn/repositories/themes.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Themes
+description: Change the color of your editor by utilizing the preset themes available in VS Code.
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+# Themes
+
+Change the color of your editor by utilizing the preset themes available in VS Code.
+
+![CodeSandbox Themes](../images/themes.jpg)
+
+## Overview
+
+CodeSandbox maps themes of Visual Studio Code extensions to apply over the editor. Currently, it only offers a set of preset themes, including various dark and light themes:
+
+- CodeSandbox (dark / light)
+- [GitHub's Default (dark / light)](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)
+- [Vesper (dark)](https://marketplace.visualstudio.com/items?itemName=raunofreiberg.vesper)
+- [Dracula (dark)](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula)
+- [poimandres (dark)](https://marketplace.visualstudio.com/items?itemName=pmndrs.pmndrs)
+
+## Configure
```

Comment:
Maybe a more specific headline, `Change theme`? might be better for SEO / Accessibility